connector/res_partner_view.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
     <data noupdate="0">
```

Comment:
You can use `<odoo noupdate="0">` directly instead of add `<data>`.
Although in this case it is not necessary to use noupdate because 0 is the default value.